### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose

Currently we are not able to merge PR in this repository as richie dependency contains error that breaks job ci. Richie 2.15.1 has been released to fix those errors.

Furthermore, django cms 3.11 has been released recently and we have updated our fork so we are now able to use this new version in our sites.

Then dependency bot warns us about a security issue with the javascript package terser so we embedded this update in this PR.

## Proposal
- [x] Upgrade to richie 2.15.1
- [x] Upgrade our django-cms fork to 3.11
- [x] Upgrade terser to 5.14.2

### CODE REVIEW

**Review by kernicPanel — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -341,18 +341,128 @@ version: 2.1
...
+                    tags:
+                        only: /ademe-.*/
+                name: test-back-ademe
+                requires:
+                    - build-back-ademe
+                site: ademe
+            - hub:
+                filters:
+                    tags:
+                        only: /^ademe-.*/
```
Comment (carofun) [inline .circleci/config.yml:397]:
```suggestion
                        only: /^ademe-.*/
```
why this filter compare to other has a ^ before?

File: .circleci/config.yml [hunk 2]
```diff
@@ -341,18 +341,128 @@ version: 2.1
...
+                    tags:
+                        only: /cnfpt-.*/
+                name: test-back-cnfpt
+                requires:
+                    - build-back-cnfpt
+                site: cnfpt
+            - hub:
+                filters:
+                    tags:
+                        only: /^cnfpt-.*/
```
Comment (carofun) [inline .circleci/config.yml:459]:
```suggestion
                        only: /^cnfpt-.*/
```
ditto

**Review by carofun — COMMENTED** [review]
commit message : upgrade => upgraded
(deps) upgrade django-cms fork to 3.11
We have **upgrade** our fork

File: .circleci/config.yml (hunk 1 above)
Comment (jbpenrath) [reply .circleci/config.yml:397]: This is regex we want only trigger this job for tag which starts with `ademe-`

Comment (carofun) [reply .circleci/config.yml:397]: I know that, but it's not written the same way for other jobs, it's written without ^, i think it's better to keep the same way of writing the rule

**Pull request merged**